### ISSUE TITLE
101 docs stale

### ISSUE THREAD

**Issue by tlambert03** [TuragaLab/chromatix#100]
just a courtesy note: 
the ["chromatix 101 docs"](https://chromatix.readthedocs.io/en/latest/101/) appear to be out of date.

```python
import chromatix.functional as cx

# We first create an empty field. 
field = cx.empty_field(shape=(512, 512), dx=1.0, spectrum=0.532, spectral_density=1.0)
```

```
AttributeError: module 'chromatix.functional' has no attribute 'empty_field'
```

**Comment by GJBoth** [comment]
Hi @tlambert03, thanks for your interest! You're right - 101 is currently broken. We had a hackathon a few weeks ago and made some changes based on the feedback we got, and we're almost finished with updating the docs - but not quite ;-)

We hope to update 101 next week. There's some small changes to the code, but the main ideas are the same. The examples in our documentation should work though! (https://chromatix.readthedocs.io/en/latest/examples/cgh/ for example).

**Comment by GJBoth** [comment]
We're working on CI for the docs so hopefully this doesn't happen anymore, but we haven't got around to it.

**Comment by tlambert03** [comment]
yep, holoscope.ipynb works great. thanks

**Comment by diptodip** [comment]
Maybe we should have put some warnings up about being in an "alpha" state :slightly_smiling_face: 

But thanks for checking things out and pointing out what was broken! I've updated the 101 and all the other examples that you tried that didn't work. We can still improve 101 to make it a better introduction to Chromatix and simulating optics in general, but it'll do for now.

A note about the parallel imaging and PSF examples: those just got updated but you'll need to make sure to either change the number of devices in the code before running the example or run on a system that has 4 GPUs.
